parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -96,12 +96,13 @@ export GPU_DEST=/usr/local/nvidia
 NVIDIA_DOCKER_VERSION=2.8.0-1
 DOCKER_VERSION=1.13.1-1
 NVIDIA_CONTAINER_RUNTIME_VERSION="3.6.0"
-export NVIDIA_DRIVER_IMAGE_TAG="${GPU_DV}-v0.0.2-rev20-sha-a0bbd5"
```

Comment:
why is this diff here? still missing a rebase? I can try and fix this if you want

Reply:
master has what's on the righ which is why it seems like some weird history issue